docs/source/cn/argsort.py

```diff
@@ -0,0 +1,42 @@
+import oneflow
+from docreset import reset_docstr
+
+reset_docstr(
+    oneflow.argsort,
+    r"""
+    argsort() -> Tensor
+
+    此运算符以指定的 dim 对输入张量进行排序，并返回排序张量的索引。
```

Comment:
```suggestion
    此算子以指定的 dim 对输入张量进行排序，并返回排序张量的索引。
```
operator 应该翻译为算子吧，后面如果有的都改一下